main.rb

```diff
@@ -0,0 +1,151 @@
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+require './book'
+# rubocop:disable all
+# Create class App
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_books
+    @books.each do |book|
+      puts "Title: \"#{book.title}\", Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+
+    person_type = gets.chomp
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      return
+    end
+
+    print 'Age: '
+
+    age = gets.chomp
+
+    print 'Name: '
+
+    name = gets.chomp
+
+    person =
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        # *If needed, ask for parameters for the option.
+        parent_permission = gets.chomp
+        parent_permission = parent_permission.downcase == 'y'
```

Comment:
- [ ] I am not able to create a student successfully, Whenever I enter the `parent_permission` I am getting and error, Kindly fix that and each person you are creating should have an `ID`

![image](https://user-images.githubusercontent.com/75126481/151421539-5135aa0e-abb5-4727-ba3d-ed542b81ad24.png)

Reply:
Thanks for your feedback